tests/geo/test_find_in_string.py

```diff
@@ -0,0 +1,388 @@
+from collections import namedtuple
+import unittest
+
+from numpy import testing
+
+from text_scrubber.geo.find_in_string import (
+    find_city_in_string,
+    find_country_in_string,
+    find_region_in_string,
+)
+
+# Match object. 'substring_range' is a tuple denoting the start and end idx of the substring in the original string.
+Match = namedtuple("Match", ["substring_range", "substring", "normalized", "score"])
+
+
+class FindCountryInStringTest(unittest.TestCase):
```

Comment:
This should use blacklist/whitelist
"University of Bern Hochschulstrasse 4 3012CH Bern Switzerland."

Reply:
Yes, but you shouldn't test two things at once. There should be a separate test for blacklist/whitelist. You can add two example, like so:
- One where `CH` is not identified, because another country is mentioned. 
- One where `CH` is identified, because another country wasn't mentioned
Then it's clear the blacklist/whitelist is at play